Controller/TemplateController.php

```diff
@@ -228,4 +230,25 @@ protected function getCurrencies($language)
 
         return $currencyTitles;
     }
+
+    /**
+     * Returns delivery states
+     *
+     * @param $language
+     * @return array
+     */
+    protected function getDeliveryStates($language)
+    {
+        $states = $this->get('sulu_product.delivery_status_manager')->findAll($language);
```

Comment:
would be nice to have a private method to get the manager like: getDeliveryStatusManager()